app/src/main/res/drawable/notification_icon.xml

```diff
@@ -1,5 +0,0 @@
-<vector android:height="24dp" android:tint="#FF0000"
```

Comment:
Could also be replaced by the colors.xml value (see comment on NotificationService.kt)